ops/Makefile

```diff
@@ -16,7 +16,7 @@ CURL_TIMEOUT?=20
 	@case $* in dev|stg|prod|prod-dmz|global) ;; *) echo "$* is not a valid environment"; exit 1;; esac
 
 api.tfvars: /dev/null
-	echo "acr_image_tag=\"$(DEPLOYED_COMMIT)\"" > $@; \
```

Comment:
Good call-out! That's definitely something we can change around. I figured this was fine as-is since it was the only image we have to deal with right now, but I can definitely see the utility in updating it.